discovery-provider/src/queries/get_ipfs_peer_info.py

```diff
@@ -0,0 +1,25 @@
+import re
+
+from src.utils.ipfs_lib import IPFSClient
+from src.utils.config import shared_config
+
+# Convert key to snake case
+pattern = re.compile(r'(?<!^)(?=[A-Z])')
+ipfs_client = IPFSClient(
+    shared_config["ipfs"]["host"], shared_config["ipfs"]["port"]
+)
+
+def convert_to_snake_case(value):
```

Comment:
i'm actually not sure if we need this or not. this is the correct way to do it for python, but this also means the response from discovery is different from content nodes. i'll leave this to your discretion

Reply:
if we look at the monitoring metrics in https://discoveryprovider.audius.co/health_check, the keys are also in snake casing. with that said, I think it's more consistent to follow the snake casing pattern

Reply:
okay, fair enough